test/attributes_test.rb

```diff
@@ -51,6 +51,14 @@ def to_h
       attributes.get('23').must_equal 'foo'
     end
 
+    it 'allows deep indifferent access' do
+      attributes = Hanami::Utils::Attributes.new(foo: {baz: 'bar'})
+      attributes.get(:foo).get(:baz).must_equal 'bar'
```

Comment:
good one, can we test against the nested string syntax instead? eg: `params.get('address.street')`